core/server/middleware/api/spam-prevention.js

```diff
@@ -54,7 +54,14 @@ spamPrevention = {
 
     // limit forgotten password requests to five requests per IP per hour for different email addresses
     // limit forgotten password requests to five requests per email address
+    // @TODO: add validation check to validation middleware
     forgotten: function forgotten(req, res, next) {
+        if (!req.body.passwordreset) {
+            return next(new errors.BadRequestError({
+                message: i18n.t('errors.api.utils.noRootKeyProvided', {docName: 'passwordreset'})
+            }));
+        }
```

Comment:
i kept this change as commit in this PR, because it's related to password reset.